ramls/oai-pmh-view.raml

```diff
@@ -49,4 +52,48 @@ resourceTypes:
           type: boolean
           required: false
           default: true
-
+  /updatedInstanceIds:
+    displayName: Stream API to get instances ids for OAI-PMH
+    type:
+      collection-get:
+       schemaCollection: oaipmhUpdatedInstanceIds
+       exampleCollection: !include examples/oaipmhinstanceids.json
+    get:
+      description: Stream updated instances ids for oai-pmh
+      is:
+        [validate]
+      queryParameters:
+        startDate:
+          description: start date
+          type: string
+          required: false
+        endDate:
+          description: start date
```

Comment:
Can you extend the description of the startDate and endDate parameters to mention what they do (filter instances? filter holdings? filter items?) and which field is used for filtering (metadata.createdDate?, metadata.updatedDate?)? What date formats are accepted (2020-01-01, 2020-01-01T00:00:00, ...)? Does endDate=2020-01-01 means the same as endDate=2020-01-01T23:59:59.999 and includes 2020-01-01T12:00:00?

Reply:
Done